### ISSUE TITLE
Binaries for management and signal (apt/yum)

### ISSUE THREAD

**Issue by alepaes1975** [netbirdio/netbird#532]
Hi,

It's a great add if the management and signal could be included to apt/yum repository to bypass the need of use Docker.
Some people (me included) prefer other alternatives like standalone installations or via LXC. This will help the installations with Ansible/Salt.

Best regards!

**Comment by bmullan** [comment]
I agree, I too would like to see those added as apt/yum repository files if possible!

**Comment by KnowWhoami** [comment]
Hey @mlsmaycon @nazarewk 
Are we moving forward with this feature? 
I see that we already create binaries for clients, mangament, signal , relay -> Except for client & client-ui, we are not packaging them as .deb/rpm and thus not been added into the repo.

**Comment by 0xMudit** [comment]
Hi @mlsmaycon  @nazarewk , I'd be interested in working on this.

Current state of the packaging config (.goreleaser.yaml): the client is already
published as deb/rpm to pkgs.wiretrustee.com via goreleaser nfpms, while
management, signal, and relay are only shipped as Docker images and raw
binaries.

Proposal:

- Add nfpms entries for management and signal (and relay, if wanted), reusing
  the existing client package patterns
- Ship systemd unit files for each service, similar to how the client ships
  its own service definition
- Publish to the same pkgs.wiretrustee.com apt/yum repos; RPM signing config
  already exists, it would just need to cover the new packages

Two things I'd need direction on from the team:

1. Should these packages manage their own users/permissions (e.g., a dedicated
   system user), or follow the Docker-based getting-started layout?
2. Any preferences on default config file locations under /etc/?

Happy to also send a documentation PR for self-hosters once the packaging
shape is agreed. Thanks!
